ci/scripts/multi-arch-docker.sh

```diff
@@ -14,42 +15,60 @@ components=(
   "compactor-node"
 )
 
+# push images to gchr
+function pushGchr() {
+  GHCRTAG="${ghcraddr}/$1:$2"
+  echo "push to gchr, image tag: ${GHCRTAG}"
+  docker manifest create --insecure "$GHCRTAG" \
+    --amend "${ghcraddr}/$1:latest-x86_64" \
+    --amend "${ghcraddr}/$1:latest-aarch64"
+  docker manifest push --insecure "$GHCRTAG"
+}
+
+# push images to dockerhub
+function pushDockerhub() {
+  if [ "$1" == "risingwave" ]; then
+    DOCKERTAG="${dockerhubaddr}:$2"
+    echo "push to dockerhub, image tag: ${DOCKERTAG}"
+    docker manifest create --insecure "$DOCKERTAG" \
+      --amend "${dockerhubaddr}:latest-x86_64" \
+      --amend "${dockerhubaddr}:latest-aarch64"
+    docker manifest push --insecure "$DOCKERTAG"
+  fi
+}
+
 echo "--- ghcr login"
 echo "$GHCR_TOKEN" | docker login ghcr.io -u "$GHCR_USERNAME" --password-stdin
 
+echo "--- dockerhub login"
+echo "$DOCKER_TOKEN" | docker login -u "risingwavelabs" --password-stdin
+
 for component in "${components[@]}"
 do
   echo "--- multi arch image create : ${component}"
   if [[ "${#BUILDKITE_COMMIT}" = 40 ]]; then
     # If the commit is 40 characters long, it's probably a SHA.
-    TAG="${ghcraddr}/${component}:git-${BUILDKITE_COMMIT}"
-    docker manifest create --insecure "$TAG" \
-      --amend "${ghcraddr}/${component}:latest-x86_64" \
-      --amend "${ghcraddr}/${component}:latest-aarch64"
-    docker manifest push --insecure "$TAG"
+    TAG="git-${BUILDKITE_COMMIT}"
+    pushGchr ${component} ${TAG}
   fi
 
   if [ "${BUILDKITE_SOURCE}" == "schedule" ] || [ "${BUILDKITE_SOURCE}" == "ui" ]; then
     # If this is a schedule/ui build, tag the image with the date.
-    TAG="${ghcraddr}/${component}:nightly-${date}"
-    docker manifest create --insecure "$TAG" \
-      --amend "${ghcraddr}/${component}:latest-x86_64" \
-      --amend "${ghcraddr}/${component}:latest-aarch64"
-    docker manifest push --insecure "$TAG"
+    TAG="nightly-${date}"
+    pushGchr ${component} ${TAG}
+    pushDockerhub ${component} ${TAG}
   fi
 
   if [[ -n "${BUILDKITE_TAG}" ]]; then
     # If there's a tag, we tag the image.
-    TAG="${ghcraddr}/${component}:${BUILDKITE_TAG}"
-    docker manifest create --insecure "$TAG" \
-      --amend "${ghcraddr}/${component}:latest-x86_64" \
-      --amend "${ghcraddr}/${component}:latest-aarch64"
-    docker manifest push --insecure "$TAG"
+    TAG="${BUILDKITE_TAG}"
+    pushGchr ${component} ${TAG}
+    pushDockerhub ${component} ${TAG}
+
+    TAG="latest"
+    pushDockerhub ${component} ${TAG}
```

Comment:
Why is this logic inside the branch `if [[ -n "${BUILDKITE_TAG}" ]]; then` ?

Reply:
Because we hope that the latest image on dockerhub should be a relatively stable release version.

Reply:
OK I get this.

Reply:
If we need recent updates, we can download nightly images.